tox.ini

```
@@ -1,13 +1,13 @@
 [tox]
-envlist = py{38}-django{22,30,31},quality
+envlist = py{38}-django{22,30,31,32},quality
```

This change shouldn't be needed right now. We can do this in a different PR.
```suggestion
envlist = py{38}-django{22,30,31},quality
```